### DISCUSSION TITLE
Save Audio in Chunks

### DISCUSSION THREAD

**Discussion by drock577** [aiortc/aiortc#976] [Q&A]
Hello I have followed the Server example in the repo. I would like to know if it is possible to save the audio stream into chunks. 
I am saving the stream with the MediaRecorder and cannot seem to find a way to switch to saving to a new audio file after a specified amount of time. 

Questions:
Is it possible to save the audio stream into chunks?
Is there an ideal way to chunk audio?

Thank you!

**Comment by gjbrito17** [comment]
did you find a solution for this? @drock577
